nuxt.config.js

```diff
@@ -21,30 +21,33 @@ export const env = {
 const meta = [
   { charset: 'utf-8' },
   {
+    hid: 'description',
     name: 'description',
     content:
-      'A new Openverse search tool for creators seeking to discover and reuse free resources with greater ease.',
+      'The search engine for openly-licensed content, including Creative Commons licensed images, audio, and more.',
```

Comment:
We should discuss and decide some definitive tagline for the project, we've used a lot of different text in a lot of different places.

Reply:
Most definitely. The challenge too is that "Creative Commons" is one of our bigger search terms. I think a meta description can be _sliiightly_ keyword-stuffy, and we can have a better tagline in the actual site content.

I'm totally open to revisions here still.